### PR TITLE
My solution to the last task - 'Baker_pete'

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/countCakes.test.js [hunk 1]
```diff
@@ -8,13 +8,42 @@ describe(`Function 'countCakes':`, () => {
   });
 
   it(`should return a number`, () => {
+    expect(typeof countCakes()).toBe('number');
+  });
 
+  it(`should return 0 if objects are empty`, () => {
+    expect(countCakes({}, {})).toBe(0);
   });
```
Comment (Oleksandra-Pasenchenko) [inline src/countCakes.test.js]: It accepts only non-empty objects, so this test is not needed

File: src/countCakes.test.js [hunk 2]
```diff
@@ -8,13 +8,42 @@ describe(`Function 'countCakes':`, () => {
...
 
-  it(`should return the number of cakes 
-    when Pete has as many ingredients as the recipe requires`, () => {
+  it(`should return the number of cakes due to ingradients`, () => {
```
Comment (Oleksandra-Pasenchenko) [inline src/countCakes.test.js]:
```suggestion
  it(`should return the number of cakes due to ingredients`, () => {
```

File: src/countCakes.test.js [hunk 3]
```diff
@@ -8,13 +8,42 @@ describe(`Function 'countCakes':`, () => {
...
+      oil: 100,
+    };
+    const available = {
+      flour: 50,
+      sugar: 100,
+    };
+
+    expect(countCakes(recipe, available)).toBe(0);
+  });
 });
```
Comment (Oleksandra-Pasenchenko) [inline src/countCakes.test.js:79]: How about adding more tests? Think about some edge cases for this task

Comment (ghost) [reply src/countCakes.test.js:79]: Thanks! I've added 2 more tests.
